index.html

```diff
@@ -63,16 +63,17 @@
               </script>
               <input id="targetUsername" type="url" class="input-block form-control" placeholder="username">
               <input id="githubPassword" type="password" class="input-block form-control" placeholder="token/password">
-              <button id="listLabelsButton" type="button" class="btn btn-block btn-secondary" data-loading-text="Loading...">List labels in this repo</button>
+              <button id="listLabelsButton" type="button" class="btn btn-block btn-secondary mb-2" data-loading-text="Loading...">List labels in this repo</button>
             </form>
 
             <div id="optionalCopyContainer" class="card bg-light card-body mb-3">
               <button type="button" class="btn btn-block btn-secondary" data-toggle="collapse" data-target="#optionalCopyStuff">Copy an existing repo's labels (Optional)</button>
               <span class="form-text text-muted">You can apply any repository's labels and then change them to meet your needs then apply to your own repository.</span>
               <div id="optionalCopyStuff" class="collapse">
                 <form class="form-inline">
-                  <input id="copyUrl" type="url" class="input-block form-control" placeholder="username:repo">
-                  <button id="copyFromRepoButton" type="button" class="btn btn-info"  data-loading-text="Copying...">Copy</button>
+                  <input id="copyUrl" type="url" class="input-block form-control mt-2" placeholder="username:repo">
+                  <button id="copyFromRepoButton" type="button" class="btn btn-info btn-block"  data-loading-text="Adding...">Add</button>
+                  <button id="cloneFromRepoButton" type="button" class="btn btn-info btn-block" data-loading-text="Cloning...">Clone</button>
```

Comment:
My only reservation is: "Add" vs. "Clone" buttons are not self explanatory. It's confusing to have both wordings. Intuitively, "Clone" means copy, but it doesn't clearly imply the "delete-all-existing-labels" aspect. 

My suggestion: Change "Clone" to "Replace all existing labels with template repo's labels" or "Delete all existing labels and add"

I don't have a solution for a **single-world** way of describing the button.

What are your thoughts?

Reply:
I agree it is confusing in its current state. We could go with your solution for the time being.

In the long run, I think it'd be better if we added a hover popover to indicate what the buttons do.

Reply:
I'm strongly in favor of **phasing out hover for critical functionalities**, because:

- There's no hover on a touchscreen. 
- Not just more people are using touchscreen tablets to do conventional computers' tasks
- More and more Windows laptops (*11 inch and above)* are equipped with large touch screens. They are essentially becoming laptop-tablet hybrids. Namely, [Windows Surface laptop/tablet line](https://www.microsoft.com/en-us/surface/business/surface-laptop-2) and [many other manufactures, including Lenovo, HP, Samsung, Dell, etc](https://www.bestbuy.com/site/shop/windows-10-touch-screen-laptops).
- Many people are using these laptop-tablet hybrids with touchscreens, styluses, and keyboards, leaving conventional mouses out of the workflow.

Reply:
The hover declaration was a simple QOL improvement on the accordion object. What **critical functionality** are you pertaining to here?

Reply:
Oh I'm sorry about my previous comment. I see what you're saying now.

We should put a pin in that.

Reply:
What I mean is: hovering can have some cute hovering effects. But any hovering actions that triggers informational pop-ups or other things should be avoid, as we are having a growing large-screen touchscreen-laptop user population.